src/context.js

```diff
@@ -32,24 +33,39 @@ const getRule = (ctx, ruleName) => {
   return result ? result[0] : null;
 };
 
-function createContext(node, sourceFile, filePath, config) {
+function createContext(node, sourceFile, filePath, config, redoclyClient) {
   const [enabledRules, allRules] = loadRuleset(config);
+  config.headers = config.headers || [];
+  config.headers = [...(config.headers || []), {
+    regexp: `https://api.${process.env.REDOCLY_DOMAIN || 'redoc.ly'}/registry.*`,
```

Comment:
this regexp is insecure, dot is treated as any symbol.

```
new RegExp(`https://api.redoc.ly/registry.*`).test('https://apisredoc.ly/registry-test') === true
```

We either should ask users to specify true regexps here which I think will be complicated to users (considering even we as developers make insecure regexps) or we should come up with a better idea